src/Symfony/Component/Notifier/Bridge/Sms77/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+7.2
+---
+
+ * Deprecate the `Sms77Transport` class, use `SevenIoTransport`instead
```

Comment:
It’s not only this class but the whole bridge that is deprecated, isn’t it?

Reply:
That's true, on my way